Removes some naked globals & an unused admin proc

Spookerton opened the pull request:
GLOBifies some scope pollution, removes some more.
so many remain ...
NUFC

now expects to merge *after* #30265 to avoid another tests conflict

SierraKomodo reviewed (dismissed):
Unit test failure:

```
232 /global/ or /static/ vars defined (expecting exactly 233)
```

Also, squoosh.

Spookerton commented:
just making sure the tests pass, then squashing

Spookerton committed:
removes some unused code, GLOBifies some globals

Spookerton committed:
update various 0/1 truths to TRUE/FALSE + cleaning

wall_mounted

simulated\s*=\s*.+$ => simulated = TRUE|FALSE

can_self_harvest

misc robot truthy stuff and comment bits

remove unused weapon lock & killswitch from robots

also some leftover vars

wiresexposed

emagged

leaves simple bots and barriers alone for now because they have funky behavior

fabricator emag behavior

tests

SierraKomodo reviewed (changes requested):
Merge conflicts.

SierraKomodo reviewed (changes requested):
AAAA merge conflicts again that didn't show until after I approved
